### PR TITLE
Version 2

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -9,72 +9,23 @@
...
-
 #=================================================
 # RETRIEVE ARGUMENTS FROM THE MANIFEST
 #=================================================
 
-domain=$YNH_APP_ARG_DOMAIN
-path_url=$YNH_APP_ARG_PATH
-is_public=$YNH_APP_ARG_IS_PUBLIC
-admin=$YNH_APP_ARG_ADMIN
 enable_terminal=$YNH_APP_ARG_ENABLE_TERMINAL
```
Comment (alexAubin) [inline scripts/install:16]:
```suggestion
```

File: scripts/install [hunk 2]
```diff
@@ -9,72 +9,23 @@
...
-
 #=================================================
 # STORE SETTINGS FROM MANIFEST
 #=================================================
-ynh_script_progression --message="Storing installation settings..." --weight=1
 
-ynh_app_setting_set --app=$app --key=domain --value=$domain
-ynh_app_setting_set --app=$app --key=path --value=$path_url
-ynh_app_setting_set --app=$app --key=admin --value=$admin
 ynh_app_setting_set --app=$app --key=enable_terminal --value=$enable_terminal
```
Comment (alexAubin) [inline scripts/install:22]:
```suggestion
```

File: scripts/install [hunk 3]
```diff
@@ -9,72 +9,23 @@
...
-ynh_app_setting_set --app=$app --key=port_hub --value=$port_hub
-
-port_http_proxy=$(ynh_find_port --port=$(($port_hub + 1)))
-ynh_app_setting_set --app=$app --key=port_http_proxy --value=$port_http_proxy
-
 #=================================================
 # INSTALL DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Installing dependencies..." --weight=23
+#REMOVEME? ynh_script_progression --message="Installing dependencies..." --weight=23
```
Comment (alexAubin) [inline scripts/install:27]:
```suggestion
ynh_script_progression --message="Installing nodejs..." --weight=3
```

File: scripts/restore [hunk 4]
```diff
@@ -10,97 +10,49 @@
...
+chown -R root: $install_dir/
+chown -R $admin: $install_dir/.venv/
 
 #=================================================
 # SPECIFIC RESTORATION
 #=================================================
 # REINSTALL DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Reinstalling dependencies..." --weight=75
+#REMOVEME? ynh_script_progression --message="Reinstalling dependencies..." --weight=75
```
Comment (alexAubin) [inline scripts/restore:28]:
```suggestion
ynh_script_progression --message="Reinstalling nodejs..." --weight=3
```

File: scripts/upgrade [hunk 5]
```diff
@@ -88,9 +42,8 @@ ynh_add_nginx_config
 #=================================================
 # UPGRADE DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Upgrading dependencies..." --weight=83
+ynh_script_progression --message="Upgrading dependencies..." --weight=20
```
Comment (alexAubin) [inline scripts/upgrade:45]:
```suggestion
ynh_script_progression --message="Upgrading nodejs..." --weight=3
```

File: scripts/upgrade [hunk 6]
```diff
@@ -104,9 +57,9 @@ python3 -m pip install pipenv
 
 if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
-	ynh_script_progression --message="Building app..." --weight=160
+	ynh_script_progression --message="Building app..." --weight=60
```
Comment (alexAubin) [inline scripts/upgrade:60]:
```suggestion
	ynh_script_progression --message="Building app..." --weight=6
```

File: tests.toml [hunk 7]
```diff
@@ -0,0 +1,16 @@
+test_format = 1.0
+
+[default]
+
+    # -------------------------------
+    # Default args to use for install
+    # -------------------------------
+
+    args.enable_terminal = true
```
Comment (alexAubin) [inline tests.toml:9]:
```suggestion
```

Not needed, this is the same as the default value in manifest

File: scripts/change_url [hunk 8]
```diff
@@ -83,29 +23,7 @@ ynh_systemd_action --service_name=$app --action="stop"
...
-	# Delete file checksum for the old conf file location
-	ynh_delete_file_checksum --file="$nginx_conf_path"
-	mv $nginx_conf_path /etc/nginx/conf.d/$new_domain.d/$app.conf
-	# Store file checksum for the new config file location
-	ynh_store_file_checksum --file="/etc/nginx/conf.d/$new_domain.d/$app.conf"
-fi
+ynh_change_url_nginx_config
 
 #=================================================
 # SPECIFIC MODIFICATIONS
```
Comment (alexAubin) [inline scripts/change_url:29]: (Can't comment the line but `$final_path` gotta be replaced with `$install_dir` a few lines below)

**Comment by ericgaspar** [comment]
!testme

**Pull request merged**